Sources/CombineXRex/@_exported.swift

```diff
@@ -0,0 +1,2 @@
+@_exported import SwiftRex
```

Comment:
Quick comment on why that's needed ?

Reply:
For CombineX it's not. I'll remove from this PR and make a new one.

The idea behind this is that you will no longer be required to import both, SwiftRex and CombineRex.
You still can "import SwiftRex" only, if all you all you need is in the core, but if you "import CombineRex" and try to use some core entity (Reducer, for example), currently you also need to import SwiftRex. This is a bit cumbersome I think.

That's also my first step toward not having a core SwiftRex product anymore, only CombineRex, RxSwiftRex and ReactiveSwiftRex, maybe even in separate repoes is SPM doesn't add support for dependencies-per-product resolution or multiple products per repo.